html/elements/meta.json

```diff
@@ -124,6 +124,469 @@
+                "firefox_android": {
+                  "version_added": "4"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": {
+                  "version_added": "10.1"
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": "3"
+                },
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror",
+                "webview_ios": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "initial-scale": {
+            "__compat": {
+              "description": "content=\"initial-scale=[value]\"",
+              "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Viewport_meta_tag#initial-scale",
+              "spec_url": "https://drafts.csswg.org/css-viewport/#viewport-meta",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": "18"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": "4"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": {
+                  "version_added": "10.1"
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": "3"
+                },
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror",
+                "webview_ios": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "interactive-widget": {
+            "__compat": {
+              "description": "content=\"interactive-widget=[value]\"",
+              "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Viewport_meta_tag#interactive-widget",
+              "spec_url": "https://drafts.csswg.org/css-viewport/#viewport-meta",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": "131"
+                },
```

Comment:
This seems to contradict what you wrote in https://github.com/mdn/browser-compat-data/issues/25311:

> [Chrome 108 shipped interactive-widget property](https://chromestatus.com/feature/6145225857171456)

Reply:
Fixed! Thanks